### PR TITLE
Add PowerShell to index.yml

### PR DESCRIPTION
**Pull request by Vingian**
Adds [powershell and powershell_core](https://github.com/bottlesdevs/dependencies/pull/251) to the index.yml and adjust the descriptions.

## Type of change
- [ ] New dependency
- [ ] Manifest fix
- [x] Other

# Was this tested using a [local repository](https://maintainers.usebottles.com/Testing)?
- [x] Yes
- [ ] No

### CODE REVIEW

File: index.yml [hunk 1]
```diff
@@ -197,6 +197,12 @@ wsh57:
 webview2:
   Description: Microsoft Edge Web View 2
   Category: Essentials
+powershell:
+  Description: PowerShell Wrapper For Wine
```
Comment (ProjectSynchro) [inline index.yml:201]: Minor nit: You may want to specify that it is a Windows PowerShell wrapper. e.g. "Windows PowerShell Wrapper for Wine"

File: index.yml [hunk 2]
```diff
@@ -197,6 +197,12 @@ wsh57:
...
+  Category: Essentials
+powershell_core:
+  Description: Microsoft PowerShell
```
Comment (ProjectSynchro) [inline index.yml:204]: Minor nit: Make sure to specify that this is PowerShell Core, there's some small diferences between this and Windows PowerShell :smile:

**Review by ProjectSynchro — COMMENTED** [review]
Just some small comments on the wording in the descriptions.

**Comment by Vingian** [comment]
> Just some small comments on the wording in the descriptions.

Thanks! Adjustments made. :thumbsup:

**Pull request merged**